plugins/terraform/terraform.go

```diff
@@ -0,0 +1,38 @@
+package terraform
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func TerraformCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Terraform CLI",
+		Runs:    []string{"terraform"},
+		DocsURL: sdk.URL("https://developer.hashicorp.com/terraform/cli"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
+		),
+		Uses: []schema.CredentialUsage{
+			{
+				Description: "Credentials to use within the Terraform project",
```

Comment:
```suggestion
				Description: "Credentials to use within terraform projects",
```

Reply:
Done, thanks!

Reply:
Hmm, I think the previous version makes more sense, to be honest. This configuration is highly dependent on the Terraform project, so the more common use case would be to set up a different profile per project.

Reply:
Okay, changed this back to normal. We could revisit this if better wording comes to mind later.